spec/helpers/application_helper_spec.rb

```diff
@@ -311,22 +317,58 @@
 
   describe 'expire_date_label_and_value' do
 
-    it 'returns date with style "yes" if expire_date more than a month away' do
-      user_payment.update(expire_date: Time.zone.today + 2.months)
-      response = /class="Yes".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
-    end
+    context 'user' do
 
-    it 'returns date with style "maybe" if expire_date less than a month away' do
-      user_payment.update(expire_date: Time.zone.today + 2.days)
-      response = /class="Maybe".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
+      it 'returns date with style "yes" if expire_date more than a month away' do
+        user_payment.update(expire_date: Time.zone.today + 2.months)
+        response = /class="Yes".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns date with style "maybe" if expire_date less than a month away' do
+        user_payment.update(expire_date: Time.zone.today + 2.days)
+        response = /class="Maybe".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns date with style "no" if expired' do
+        user_payment.update(expire_date: Time.zone.today - 1.day)
+        response = /class="No".*#{user_payment.expire_date}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
+
+      it 'returns tooltip explaining expiration date' do
+        user_payment.update(expire_date: Time.zone.today)
+        response = /#{t('users.show.membership_expire_date_tooltip')}/
+        expect(expire_date_label_and_value(user)).to match response
+      end
     end
 
-    it 'returns date with style "no" if expired' do
-      user_payment.update(expire_date: Time.zone.today - 1.day)
-      response = /class="No".*#{user_payment.expire_date}/
-      expect(expire_date_label_and_value(user)).to match response
+    context 'company' do
+
+      it 'returns date with style "yes" if expire_date more than a month away' do
+        brand_payment.update(expire_date: Time.zone.today + 2.months)
+        response = /class="Yes".*#{brand_payment.expire_date}/
+        expect(expire_date_label_and_value(company)).to match response
+      end
+
```

Comment:
What is supposed to happen when expire_date == 1 month away?  (either change the wording in the "it.." statements to have one of them include '=' or include another test for that specific case)